integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -49,8 +49,8 @@ fn print_and_assert_shard_accounts(client: &Client) {
         let mut shard_accounts = vec![];
         for item in trie.lock_for_iter().iter().unwrap() {
             let (key, value) = item.unwrap();
-            let state_record = StateRecord::from_raw_key_value(key, value).unwrap();
-            if let StateRecord::Account { account_id, .. } = state_record {
+            let state_record = StateRecord::from_raw_key_value(key, value);
+            if let Some(StateRecord::Account { account_id, .. }) = state_record {
```

Comment:
This is an actual change in semantics, is that intentional?

Reply:
Intentional, explained in the commit description